### ISSUE TITLE
Can't use on angular

### ISSUE THREAD

**Issue by Freir96** [tiagoporto/accessibility-buttons#25]
**Describe the bug**
I have imported accessibilityButtons as shown in the README, and it didn't casue any error:
`import { accessibilityButtons } from 'accessibility-buttons/dist/js/accessibility-buttons';`
But Calling accessibilityButtons() results in error `Object(...) is not a function`.
If this library does not work on angular please update your README. 

**To Reproduce**
1. Create Angular project
2. import accessibilityButtons in a component
3. call accessibilityButtons()

**Expected behavior**
The accessibilityButtons should be initiated

**Enviroment (please complete the following information):**
 - Windows
 - Opera

** Error message
  ```
main.ts:12 TypeError: Object(...) is not a function
    at new AppComponent (app.component.ts:42)
    at createClass (core.js:24584)
    at createDirectiveInstance (core.js:24400)
    at createViewNodes (core.js:35006)
    at createRootView (core.js:34878)
    at callWithDebugContext (core.js:36407)
    at Object.debugCreateRootView [as createRootView] (core.js:35644)
    at ComponentFactory_.create (core.js:23542)
    at ComponentFactoryBoundToModule.create (core.js:19549)
    at ApplicationRef.bootstrap (core.js:32042)
  ```

**Comment by tiagoporto** [comment]
Thanks @Freir96, I'll take a look

**Comment by tiagoporto** [comment]
Finally, after all these years. I've refactored this lib.
Now it's using web components, It should work with Angular.
[v5.0.0](https://github.com/tiagoporto/accessibility-buttons/releases/tag/accessibility-buttons%405.0.0
)
